graphrag_sdk/steps/extract_data_step.py

```
@@ -41,28 +41,41 @@
         ontology: Ontology,
         model: GenerativeModel,
         graph: Graph,
-        config: dict = {
+        config: Optional[dict] = {
             "max_workers": 16,
             "max_input_tokens": 500000,
             "max_output_tokens": 8192,
         },
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)